### DISCUSSION TITLE
rel boolean type is not working with remark-external-links

### DISCUSSION THREAD

**Discussion by MejanH** [remarkjs/remark#533] [General] [answered]
`rel: false` option not working.

`Type 'boolean' is not assignable to type 'string | string[]'.ts(2322)
index.d.ts(30, 5): The expected type comes from property 'rel' which is declared here on type 'RemarkExternalLinksOptions'`

**Comment by ChristianMurphy** [comment, marked as answer]
@MejanH can you clarify what you mean by "not working"?

Is it just the typescript error?
If so a PR to add false to this type union would be welcome: https://github.com/remarkjs/remark-external-links/blob/2c6fc4bd9801489809789964dff96c9ecf6e0605/types/index.d.ts#L18-L30

Or does it have an issue during runtime?

**Reply by MejanH** [reply]
I just made a **pull request** now. after adding `boolean` type, it's working.

**Reply by ChristianMurphy** [reply]
Thanks!
For posterity the PR is https://github.com/remarkjs/remark-external-links/pull/20

**Reply by MejanH** [reply]
My please brother, This is my second time making a pull request. Thank you for giving me the opportunity.
